agr_literature_service/api/triggers/reference_sql_func_triggers.py

```diff
@@ -11,15 +11,42 @@
   END;
 $$;
 """
-reference_update_trigger = """
-DROP TRIGGER IF EXISTS reference_citation_trigger on public.reference;
-CREATE TRIGGER reference_citation_trigger
-AFTER UPDATE OR INSERT ON reference
-        FOR EACH ROW
-        EXECUTE FUNCTION public.reference_update_citation();
```

Comment:
Thanks both for looking over the trigger code! @valearna @azurebrd

Good question! @azurebrd 
Since current version only handle these two cases (see attached) so I was assuming so.

Reply:
cool, thanks @sweng66 !